README.md

```diff
@@ -133,6 +133,36 @@ int main(void){
 }
 ```
 
+## Enabling traces in mbed OS 5
+
+To enable traces in mbed OS 5, add the following to your ``mbed_app.json``:
+
+```json
+{
+   "target_overrides": {
+      "*": {
+         "mbed-trace.enable": 1
```

Comment:
Checked to be sure, and it does need to be added. (I've seen some things compile without explicitly adding the relevant feature into the build, like network-socket API using code without including FEATURE_IPV4)

Anyway, `"target.features_add": ["COMMON_PAL"],` is missing here.